### PR TITLE
docs: add detection task to readme

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Add detection task to readme

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by williamFalcon** [comment]
@kaushikb11 is the task to detect images or to detect objects?
If so, can this be named ObjectDetection?

File: README.md [hunk 1]
```diff
@@ -275,6 +275,50 @@ Flash has a TabularClassification task to tackle any tabular classification prob
...
+<details>
+  <summary>View example</summary>
+
+  To illustrate, say we want to build a model on a tiny coco dataset.
+
+  ```python
+  # import our libraries
+  import flash
+  from flash.core.data import download_data
+  from flash.vision import ImageDetectionData, ImageDetector
```
Comment (williamFalcon) [inline README.md:291]: Shouldn't this be ObjectDetector?

**Comment by kaushikb11** [comment]
> @kaushikb11 is the task to detect images or to detect objects?
> If so, can this be named ObjectDetection?

Yup, `ObjectDetect*` would be better. Pushing the changes now.

**Comment by williamFalcon** [comment]
ObjectDetection* :)

**Comment by kaushikb11** [comment]
> ObjectDetection* :)
> 
> ObjectDetect doesn't have the same grammatical meaning

Oh, I meant like `ObjectDetector` / `ObjectDetectionData`.

File: README.md (hunk 1 above)
Comment (kaushikb11) [reply README.md]: Resolved!

**Pull request merged**